app/controllers/users_controller.rb

```diff
@@ -1,4 +1,12 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
```

Comment:
dùng except thay vì only trong trường hợp này sẽ gọn hơn
```suggestion
  before_action :logged_in_user, except: %i(show new create)
```